fiftyone/core/odm/database.py

```diff
@@ -147,7 +147,7 @@ def establish_db_conn(config):
         _connection_kwargs["port"] = int(established_port)
     if config.database_uri is not None:
         _connection_kwargs["host"] = config.database_uri
-    elif _db_service is None:
+    if _db_service is None:
```

Comment:
Revert this! It's unrelated to this PR and is also not what we want (would cause a mongo service to start even if user has `database_uri` configured)